### ISSUE TITLE
Can i Use 80mm Paper?

### ISSUE THREAD

**Issue by ryandan29** [paystory-de/thermal-printer-cordova-plugin#36]

**Comment by saaiful** [comment]
Yes, But you need to use `https://github.com/saaiful/thermal-printer-cordova-plugin` this version.
to add `nm add https://github.com/saaiful/thermal-printer-cordova-plugin `

**Comment by saaiful** [comment]
Now you can use `data.printerWidthMM` using this plugin (JS&TS).
